doc/Developing.md

```diff
@@ -50,7 +50,16 @@ tab with the frontend.
 
 ## Run the electron-based example application
 
-It can also be started with:
+From the root directory run:
+
+    npm run rebuild:electron
+
+This command rebuilds native node packages against the version of node used by electron.
+To rollback changes run:
+
+    npm run rebuild:web
+
+It can also be started with from examples/electron directory:
```

Comment:
I don't see why someone would want to rollback however ? 
Is this more confusing then helping?

Maybe move this to the web basec example section?